frontend/lib/src/components/widgets/AudioInput/formatTime.ts

```diff
@@ -15,12 +15,13 @@
  */
 
 const formatTime = (timeMs: number): string => {
-  const date = new Date(timeMs)
+  const totalSeconds = Math.floor(timeMs / 1000)
+  const minutes = Math.floor(totalSeconds / 60)
+  const seconds = totalSeconds % 60
 
-  return date.toLocaleTimeString(undefined, {
-    minute: "2-digit",
-    second: "2-digit",
-  })
+  return `${minutes.toString().padStart(2, "0")}:${seconds
+    .toString()
+    .padStart(2, "0")}`
 }
```

Comment:
**suggestion (non-blocking):** Another way to solve this is to specify a specific locale and timezone:

```tsx
const formatTime = (timeMs: number): string => {
  const date = new Date(timeMs)

  return date.toLocaleTimeString("en-US", {
    minute: "2-digit",
    second: "2-digit",
    timeZone: "UTC",
  })
}
```

The solution + tests as written in the PR already handle the job as needed so no need to take this suggestion, but wanted to provide this for completeness since the original code is actually directionally correct.

Reply:
for posterity, gonna do this and leave the locale param blank so that it formats it appropriately. the UTC should take care of eliminating any :30 or :45 strangeness